CONTRIBUTING.md

```diff
@@ -98,9 +98,9 @@ meets these guidelines:
 7.  If you are asked to update your pull request with some changes there's
     no need to create a new one. Push your changes to the same branch.
 
-## Local development
+## Setup Local Environment for Development
```

Comment:
@kristw just noticed that changing the header breaks the table of contents above